x-pack/plugins/infra/server/lib/alerting/inventory_metric_threshold/evaluate_condition.ts

```diff
@@ -51,14 +52,18 @@ export const evaluateCondition = async ({
   compositeSize: number;
   filterQuery?: string;
   lookbackSize?: number;
+  startTime?: number;
 }): Promise<Record<string, ConditionResult>> => {
   const { comparator, warningComparator, metric, customMetric } = condition;
   let { threshold, warningThreshold } = condition;
 
+  const to = startTime ? moment(startTime) : moment();
```

Comment:
@simianhacker correct me if I am wrong. I would call it endTime, because looking at the test where you make use of it, it refers to the hosts_only.max (the 2nd timestamp). So for me `from` is the `startTime` and `to` is the `endTime`. 

This naming makes more sense to me https://github.com/elastic/kibana/blob/main/x-pack/plugins/infra/public/metrics_overview_fetchers.test.ts#L64